.travis.yml

```diff
@@ -14,5 +15,8 @@ addons:
     packages:
     - git-svn
 
+before_install:
+  gem update --system
```

Comment:
This is a workaround for a problem with Bundler 1.16.1 and Rubygems 2.7.3 (see https://github.com/travis-ci/travis-ci/issues/8978#issuecomment-354036443)

Reply:
Please add this comment into the file.